pkg/controller/certificaterequests/acme/acme.go

```diff
@@ -206,20 +206,12 @@ func (a *ACME) Sign(ctx context.Context, cr *v1.CertificateRequest, issuer v1.Ge
 		return nil, a.acmeClientV.Orders(order.Namespace).Delete(ctx, order.Name, metav1.DeleteOptions{})
 	}
 
-	bundle, err := pki.ParseSingleCertificateChainPEM(order.Status.Certificate)
-	if err != nil {
-		log.Error(err, "failed to successfully build a certificate chain from data on Order resource.")
-		return nil, a.acmeClientV.Orders(order.Namespace).Delete(ctx, order.Name, metav1.DeleteOptions{})
-	}
-
 	log.V(logf.InfoLevel).Info("certificate issued")
 
 	// Order valid, return cert. The calling controller will update with ready if its happy with the cert.
 	return &issuerpkg.IssueResponse{
-		Certificate: bundle.ChainPEM,
-		CA:          bundle.CAPEM,
```

Comment:
Is this because we don't get the actual root back from ACME servers, so removing the intermediate and putting it in CA may not be correct?

Reply:
That's right. Although this works in the testing I did, I can't test every user's working setup. Best to leave the current behavior to not break anyone.